src/ripple/protocol/impl/TER.cpp

```diff
@@ -58,6 +64,7 @@ transResults()
         MAKE_ERROR(tecOVERSIZE,                      "Object exceeded serialization limits."),
         MAKE_ERROR(tecUNFUNDED,                      "Not enough XRP to satisfy the reserve requirement."),
         MAKE_ERROR(tecUNFUNDED_ADD,                  "DEPRECATED."),
+        MAKE_ERROR(tecAMM_UNFUNDED,                  "Insufficient balance to fund AMM."),
```

Comment:
Suggestion: rename this to `tecUNFUNDED_AMM` to fit the pattern of the other UNFUNDED ter codes.